src/write/segy_write_append.jl

```diff
@@ -0,0 +1,16 @@
+export segy_write_append
+
+function segy_write_append(file::String, block::SeisBlock)
+
+    # Open buffer for writing
+    s = open(file, "r+")
+
+
+    # Write Data
+    ns,ntraces = size(block.data)
```

Comment:
probably need to check it's the same as in the rest of the existing file.

Reply:
> probably need to check it's the same as in the rest of the existing file.

Add a new function check_fileheader.jl to guarantee that they are same.